tools/xds-client/main.go

```diff
@@ -103,7 +103,8 @@ func newRunCmd() *cobra.Command {
 					<-time.After(delay)
 					// proceed
 
-					errCh <- func() (errs error) {
+					errCh <- func() error {
+						var errs error
```

Comment:
I don't think this is correct anymore, `errs` is used in deferred functions here but now it's never set whereas before with the named return, it's set to the returned error.

Reply:
Such `defers` may be present at other places in this PR, I'll take another look tomorrow.

Reply:
Added a commit to change these back. Not quite sure how/whether it's worth getting rid of them for `defer`.